From 7bb5810e38f8430e0bc0bc994ae47b75326191ae Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Wed, 29 May 2019 09:32:17 +0800
Subject: [PATCH] samba: enable samba_export_all_rw and samba_export_all_ro

Fix:
type=AVC msg=audit(1559093196.731:71): avc:  denied  { search } for
pid=230 comm="nmbd" name="/" dev="tmpfs" ino=11008
scontext=system_u:system_r:nmbd_t:s0-s15:c0.c1023
tcontext=system_u:object_r:tmpfs_t:s0 tclass=dir permissive=0

type=AVC msg=audit(1559093196.733:72): avc:  denied  { create } for
pid=230 comm="nmbd" name="msg.lock"
scontext=system_u:system_r:nmbd_t:s0-s15:c0.c1023
tcontext=system_u:object_r:samba_var_t:s0 tclass=dir permissive=0

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/services/samba.te | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/modules/services/samba.te b/policy/modules/services/samba.te
index 3d4bda4..b9c775f 100644
--- a/policy/modules/services/samba.te
+++ b/policy/modules/services/samba.te
@@ -62,7 +62,7 @@ gen_tunable(samba_enable_home_dirs, false)
 ##	any content read only.
 ##	</p>
 ## </desc>
-gen_tunable(samba_export_all_ro, false)
+gen_tunable(samba_export_all_ro, true)
 
 ## <desc>
 ##	<p>
@@ -70,7 +70,7 @@ gen_tunable(samba_export_all_ro, false)
 ##	content readable and writable.
 ##	</p>
 ## </desc>
-gen_tunable(samba_export_all_rw, false)
+gen_tunable(samba_export_all_rw, true)
 
 ## <desc>
 ##	<p>
-- 
2.7.4

